workflows/prognostic_c48_run/emulation/emulation/_monitor/__init__.py

```diff
@@ -0,0 +1,13 @@
+import os
+
+from .monitor import Config
+from .._utils import wrap_configurable_hook
+
+try:
+    config = Config.from_environ(os.environ)
+    error = None
+except Exception as e:
+    config = None
+    error = e
+
+store = wrap_configurable_hook(config, error, "store")
```

Comment:
I'm not sure `wrap_configurable_hook` is a well-factored function. The name is pretty opaque and the interface is tightly coupled to the try-except block. Seems like this would be simpler (likewise for the code in `microphysics`).

```suggestion
try:
    config = Config.from_environ(os.environ)
    store = config.store
except KeyError as e:
    logging.exception(e)
    def store(_):
        pass
```

Similar LOC but without the hard to understand function name.